### PR TITLE
Bump: Sentry Cocoa & Android

### PR DESCRIPTION
**Pull request by ueman**
## :scroll: Description
Update Sentry Cocoa and Android to the newest versions.

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] I updated the docs if needed
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Unreleased
 
 * Feat: Collect more information for exceptions collected via `FlutterError.onError` (#538)
+* Bump: Sentry-Cocoa to 7.1.4
```
Comment (ueman) [inline CHANGELOG.md]:
```suggestion
* Bump: Sentry-Cocoa to 7.1.4 (#551)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+* Bump: Sentry-Android to 5.1.0-beta.9
```
Comment (ueman) [inline CHANGELOG.md]:
```suggestion
* Bump: Sentry-Android to 5.1.0-beta.9 (#551)
```

File: flutter/android/build.gradle [hunk 3]
```diff
@@ -56,6 +56,6 @@ android {
 }
 
 dependencies {
-    api 'io.sentry:sentry-android:5.1.0-beta.6'
+    api 'io.sentry:sentry-android:5.1.0-beta.9'
```
Comment (marandaneto) [inline flutter/android/build.gradle]: going to release 5.1.0 today probably, we can wait for it

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 # Unreleased
 
 * Feat: Collect more information for exceptions collected via `FlutterError.onError` (#538)
+* Bump: Sentry-Cocoa to 7.1.4 (#551)
+* Bump: Sentry-Android to 5.1.0-beta.9 (#551)
```
Comment (marandaneto) [inline CHANGELOG.md]:
```suggestion
* Bump: Sentry-Android to 5.1.0 (#551)
```

File: flutter/android/build.gradle (hunk 3 above)
Comment (marandaneto) [reply flutter/android/build.gradle]:
```suggestion
    api 'io.sentry:sentry-android:5.1.0'
```

**Comment by marandaneto** [comment]
> 12:00 +0 -1: compiling /Users/runner/work/sentry-dart/sentry-dart/flutter/test/native_sdk_integration_test.dart [E]                                                                                    
>   TimeoutException after 0:12:00.000000: Test timed out after 12 minutes.
>   package:test_api/src/backend/invoker.dart 318:28  Invoker._handleError.<fn>
>   dart:async/zone.dart 1346:47                      _rootRun
>   dart:async/zone.dart 1258:19                      _CustomZone.run
>   package:test_api/src/backend/invoker.dart 316:10  Invoker._handleError
>   package:test_api/src/backend/invoker.dart 272:9   Invoker.heartbeat.<fn>.<fn>
>   dart:async/zone.dart 1354:13                      _rootRun
>   dart:async/zone.dart 1258:19                      _CustomZone.run
>   package:test_api/src/backend/invoker.dart 271:38  Invoker.heartbeat.<fn>
>   dart:async-patch/timer_patch.dart 18:15           Timer._createTimer.<fn>
>   dart:isolate-patch/timer_impl.dart 395:19         _Timer._runTimers
>   dart:isolate-patch/timer_impl.dart 426:5          _Timer._handleMessage
>   dart:isolate-patch/isolate_patch.dart 184:12      _RawReceivePortImpl._handleMessage

looks like similar to (#500)

**Review by marandaneto — APPROVED** [review]

**Pull request merged**